### PR TITLE
Prepare release 18.0.0

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -10,19 +10,19 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Changed
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: This should go into a separate commit.

Comment (mstajdohar) [reply docs/CHANGELOG.rst]: done. please review again
